build/shade/_k-standard-goals.shade

```diff
@@ -151,24 +150,11 @@ default SAMPLES_PROJECT_GLOB = "samples/*/*.csproj"
               }
           }
       });
-
-      projectGlobs.AddRange(srcProjects);
-    }
-    if (!BuildSrcOnly && Directory.Exists("test"))
-    {
-        projectGlobs.AddRange(Files.Include(TEST_PROJECT_GLOB).ToList());
-    }
-    if (!BuildSrcOnly && Directory.Exists("samples") && Directory.EnumerateFiles("samples", "*.csproj", SearchOption.AllDirectories).Any())
-    {
-        projectGlobs.AddRange(Files.Include(SAMPLES_PROJECT_GLOB).ToList());
     }
 
-    if (projectGlobs.Any())
+    foreach (var sln in Files.Include(Path.Combine(BASE_DIR, "*.sln")))
```

Comment:
We could also just invoke "dotnet build" and let the CLI find the .sln file to use. If there are more than one files matching `*.*proj` or `*.sln` it will fail on its own.